### PR TITLE
Fix for the creation of LSF resource settings file

### PR DESCRIPTION
**Pull request by pmb59**

### CODE REVIEW

File: Snakefile-sorting-hat [hunk 1]
```diff
@@ -311,10 +311,13 @@ rule produce_reprocess_call:
         mkdir -p {wildcards.accession}/logs
         exec &> {log}
         pushd {input.working_directory}
-        if ! [ -z {params.lsf_config} ]; then
+        if [ -f {params.lsf_config} ]; then
             if [ ! -f "lsf.yaml" ]; then
                 ln -sf {params.lsf_config} lsf.yaml
```
Comment (anilthanki) [inline Snakefile-sorting-hat:316]: What happens in case `lsf.yaml` does not exist

Comment (pmb59) [reply Snakefile-sorting-hat:316]: Do you mean the source `{params.lsf_config}` or the local one ?

Comment (anilthanki) [reply Snakefile-sorting-hat:316]: Ignore this...

**Review by anilthanki — APPROVED** [review]
LGTM

**Pull request merged**